Group-6.cpp

```diff
@@ -0,0 +1,126 @@
+#include <iostream>
+#include <stdlib.h>
+#include <string.h>
+using namespace std;
+
+struct Departments
+{
+    char deptID[20];
+    char deptName[30];
+    char deptHead[20];
+    char collegeName[30];
+};
+
+void addDept(Departments *dept, int index, int total);
+void Display(Departments dept[], int total);
+void Search(Departments dept[], int total, char value[]);
+
+int main()
+{
+    cout << "\t*************Simple College Management System*************\n";
+    cout << "\n\n";
+    cout << "\t\t\tWelcome to our program!\n\n";
+    system("pause");
+    Departments deptList[50];
+    int a = 0;
+    int total = 0;
+    int option;
+start:
+    cout << "\nWhat would you like to do:\n";
+    cout << "\t[1] Add Department\n";
+    cout << "\t[2] Display all Departments\n";
+    cout << "\t[3] Search\n";
+    cout << "\t[4] Exit\n";
+    cin >> option;
+    switch (option)
+    {
+    case 1:
+        cout << "How many Department would you like to add? ";
+        cin >> a;
+        addDept(deptList, a, total);
+        total += a;
+        break;
+    case 2:
+        if (total == 0)
+            cout << "No Department saved\n";
+        else
+        {
+            Display(deptList, total);
+            system("pause");
+        }
+        system("pause");
+        system("clear");
+        break;
+    case 3:
+        if (total == 0)
+            cout << "No Department saved\n";
+        else
+        {
+            char searchVal[20];
+            cout << "Search value: ";
+            cin >> searchVal;
+            Search(deptList, total, searchVal);
+            cout << endl;
+        }
+        system("pause");
+        system("clear");
+        break;
+    case 4:
+        cout << "Thank you";
+        exit(0);
+    default:
+        cout << "invalid value!";
+        break;
+    }
+    goto start;
+
+    return 0;
+}
+
+void addDept(Departments *dept, int index, int total)
```

Comment:
Fix variable naming on the for loop use index instead of i other than that the add function is good